package/kernel/linux/modules/netsupport.mk

```diff
@@ -1297,13 +1297,21 @@ define KernelPackage/rxrpc
   HIDDEN:=1
   KCONFIG:= \
 	CONFIG_AF_RXRPC \
-	CONFIG_RXKAD=m \
+	CONFIG_AF_RXRPC_IPV6=y \
+	CONFIG_RXKAD \
 	CONFIG_AF_RXRPC_DEBUG=n
   FILES:= \
 	$(LINUX_DIR)/net/rxrpc/rxrpc.ko
-  AUTOLOAD:=$(call AutoLoad,30,rxrpc.ko)
-  DEPENDS:= +kmod-crypto-manager +kmod-crypto-pcbc +kmod-crypto-fcrypt \
-    +kmod-udptunnel4 +kmod-udptunnel6
+  AUTOLOAD:=$(call AutoLoad,30,rxrpc)
+  DEPENDS:= \
```

Comment:
Are these kernel version dependent?

Reply:
I don't believe so, no.  Some of these are not required in the sense of linkage dependencies (I believe only the `udptunnel` ones are strictly required), but are required in order to make authenticated and encrypted connections (the "rxkad" cryptosystem involves hmac-md5 and fcrypt-pcbc; see https://datatracker.ietf.org/doc/html/draft-kaduk-afs3-rxkad-k5-kdf-00).

Reply:
If you'd prefer that `crypto-user` weren't a dependency here but rather were a dependency of the userspace program(s) to use rxrpc, I could do that instead.

Reply:
That would be a good idea, do that and I will merge it

Reply:
Great, done.  Thanks!